tests/js/server/aql/aql-geo.js

```diff
@@ -182,20 +208,68 @@ function geoSuite(isSearchAlias, analyzerType) {
       withIndex.ensureIndex({type: "geo", geoJson: true, fields: ["geo"]});
       withView = db._create(collWithView);
       let analyzers = require("@arangodb/analyzers");
+      let a = analyzers.save("geo_json", "geojson", {}, ["frequency", "norm", "position"]);
```

Comment:
Instead of "geojson" was `anaylyzerType`

Reply:
Fixed